easybuild/easyconfigs/p/pyiron/pyiron-0.5.1-foss-2023a.eb

```diff
@@ -0,0 +1,120 @@
+easyblock = 'PythonBundle'
+
+name = 'pyiron'
+version = '0.5.1'
+
+homepage = 'https://github.com/pyiron/pyiron'
+description = "An integrated development environment (IDE) for computational materials science."
+
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+builddependencies = [('poetry', '1.5.1')]
+dependencies = [
+    ('Python', '3.11.3'),
+    ('SciPy-bundle', '2023.07'),
+    ('Python-bundle-PyPI', '2023.06'),
+    ('matplotlib', '3.7.2'),
+    ('h5py', '3.9.0'),
+    ('PyYAML', '6.0'),
+    ('phonopy', '2.20.0'),
+    ('ASE', '3.22.1'),
+    ('PyTables', '3.8.0'),
+    ('nglview', '3.0.8'),
+    ('tqdm', '4.66.1'),
+    ('molmod', '1.4.8'),
+    ('scikit-learn', '1.3.1'),
+    ('yaff', '1.6.0'),
+    ('TAMkin', '1.2.6'),
+    ('QuickFF', '2.2.7'),
+    ('plotly.py', '5.16.0'),
+    ('GitPython', '3.1.40'),
+    ('Greenlet', '3.0.2'),
+    ('dill', '0.3.7'),
+    ('sympy', '1.12'),
+    ('pymatgen', '2023.12.18'),
+]
+
+check_ldshared = True
+use_pip = True
+
+exts_list = [
+    ('pint', '0.23', {
+        'source_tmpl': 'Pint-%(version)s.tar.gz',
+        'checksums': ['e1509b91606dbc52527c600a4ef74ffac12fff70688aff20e9072409346ec9b4'],
+    }),
+    ('h5io', '0.1.2', {
+        'checksums': ['6400543224c489f1cf487be551f892310fa7060fcd4935b84e515efdc1e1fa43'],
+    }),
+    ('defusedxml', '0.6.0', {
+        'checksums': ['f684034d135af4c6cbb949b8a4d2ed61634515257a67299e5f940fbaa34377f5'],
+    }),
+    ('pysqa', '0.1.7', {
+        'checksums': ['fdc37e0178649750e7386b4e9b8287693ee2d47559723c11b9ad42afcfc55900'],
+    }),
+    ('SQLAlchemy', '2.0.23', {
```

Comment:
-> Dependency? SQLAlchemy-2.0.25-GCCcore-12.3.0.eb

Reply:
Fixed, that seems to also be where dependency on Greenlet comes from, so that should be pulled in automatically.